packager/live_packager_test.cc

```diff
@@ -942,4 +942,89 @@ INSTANTIATE_TEST_CASE_P(
             5, "audio/en/init.mp4", LiveConfig::EncryptionScheme::SAMPLE_AES,
             LiveConfig::OutputFormat::TS, LiveConfig::TrackType::AUDIO,
             "audio/en/%05d.m4s", false}));
+
+struct TimedTextTestCase {
+  const char* media_segment_format;
+  LiveConfig::TrackType track_type;
+  LiveConfig::OutputFormat output_format;
+  Status expected_status;
+};
+
+class TimedTextParameterizedTest
+    : public LivePackagerBaseTest,
+      public ::testing::WithParamInterface<TimedTextTestCase> {
+  void SetUp() override {
+    LivePackagerBaseTest::SetUp();
+
+    LiveConfig live_config;
+    live_config.format = GetParam().output_format;
+    live_config.track_type = GetParam().track_type;
+    live_config.protection_scheme = LiveConfig::EncryptionScheme::NONE;
+    SetupLivePackagerConfig(live_config);
+  }
+};
+
+TEST_P(TimedTextParameterizedTest, VerifyTimedText) {
```

Comment:
From this test, I don't see where are we getting a sequence number and decode time for mp4 ?

Reply:
As in add a check for mp4 segments like we do in this test? https://github.com/Pluto-tv/shaka-packager-live/blob/pluto-cmake/packager/live_packager_test.cc#L779